### PR TITLE
Maintain `FunctorFilter` documents

### PR DESCRIPTION
**Pull request by hadilq**
This PR completes some TODO of `FunctorFilter`'s `README.md` file.

### CODE REVIEW

File: arrow-docs/docs/arrow/typeclasses/functorfilter/README.md [hunk 1]
```diff
@@ -9,6 +9,50 @@ permalink: /arrow/typeclasses/functorfilter/
 
 
 
+`FunctorFilter` is a typeclass that abstracts over sequential execution of code.
```
Comment (pakoito) [inline arrow-docs/docs/arrow/typeclasses/functorfilter/README.md:12]: The tagline is copied from Monad I believe :D

File: arrow-docs/docs/arrow/typeclasses/monad/README.md [hunk 2]
```diff
@@ -148,4 +148,4 @@ import arrow.typeclasses.Monad
 TypeClass(Monad::class).hierarchyGraph()
 ```
 
-[monad_law_source]: https://github.com/arrow-kt/arrow-core/blob/master/arrow-core-test/src/main/kotlin/arrow/test/laws/MonadLaws.kt
+[monad_law_source]: https://github.com/arrow-kt/arrow-core/blob/master/arrow-core-test/src/main/kotlin/arrow/core/test/laws/MonadLaws.kt
```
Comment (pakoito) [inline arrow-docs/docs/arrow/typeclasses/monad/README.md:151]: Thanks!

File: arrow-docs/docs/arrow/typeclasses/functorfilter/README.md (hunk 1 above)
Comment (hadilq) [reply arrow-docs/docs/arrow/typeclasses/functorfilter/README.md:12]: Yes! :D Everything is copy/paste.

Comment (pakoito) [reply arrow-docs/docs/arrow/typeclasses/functorfilter/README.md:12]: I mean, it needs to be updated for this typeclass. It's what's left to go to production 🙏

**Comment by pakoito** [comment]
Just one fix to the typeclass description, as it's right now the same as monad. Then I can merge it!

**Pull request merged**